custom_components/car_wash/binary_sensor.py

```diff
@@ -185,7 +197,7 @@ async def async_update(self):
             fc_date = fcast.get(ATTR_FORECAST_TIME)
             if isinstance(fc_date, int):
                 fc_date = dt_util.as_local(
-                    datetime.utcfromtimestamp(fc_date / 1000)
+                    datetime.datetime.fromtimestamp(fc_date, tz=datetime.timezone.utc)
```

Comment:
What are these changes for?  What problem do they solve?

Reply:
No Problem, but was "request" via vscode Extension probably.
Documentation: https://docs.python.org/3/library/datetime.html#datetime.datetime.utcfromtimestamp

Deprecated since version 3.12: Use [datetime.fromtimestamp()](https://docs.python.org/3/library/datetime.html#datetime.datetime.fromtimestamp) with [UTC](https://docs.python.org/3/library/datetime.html#datetime.UTC) instead.

Reply:
Ok. Thanks